src/nunit-gui/Model/ResultSummaryReporter.cs

```diff
@@ -31,57 +30,43 @@ public static class ResultSummaryReporter
     {
         public static string WriteSummaryReport(ResultSummary summary)
         {
-            var sb = new StringBuilder();
-            using (var writer = new StringWriter(sb))
-            {
-                writer.WriteLabelLine("Overall result: ", summary.OverallResult);
-
-                writer.WriteSummaryCount("  Test Count: ", summary.TestCount);
-                writer.WriteSummaryCount(", Passed: ", summary.PassCount);
-                writer.WriteSummaryCount(", Failed: ", summary.FailedCount/*, ColorStyle.Failure*/);
-                writer.WriteSummaryCount(", Warnings: ", summary.WarningCount/*, ColorStyle.Warning*/);
-                writer.WriteSummaryCount(", Inconclusive: ", summary.InconclusiveCount);
-                writer.WriteSummaryCount(", Skipped: ", summary.TotalSkipCount);
-                writer.WriteLine();
-
-                if (summary.FailedCount > 0)
-                {
-                    writer.WriteSummaryCount("    Failed Tests - Failures: ", summary.FailureCount);
-                    writer.WriteSummaryCount(", Errors: ", summary.ErrorCount/*, ColorStyle.Error*/);
-                    writer.WriteSummaryCount(", Invalid: ", summary.InvalidCount);
-                    writer.WriteLine();
-                }
-                if (summary.TotalSkipCount > 0)
-                {
-                    writer.WriteSummaryCount("    Skipped Tests - Ignored: ", summary.IgnoreCount);
-                    writer.WriteSummaryCount(", Explicit: ", summary.ExplicitCount);
-                    writer.WriteSummaryCount(", Other: ", summary.SkipCount);
-                    writer.WriteLine();
-                }
+            var writer = new StringBuilder();
+            writer.AppendLine($"Overall result: {summary.OverallResult}");
 
-                writer.WriteLabelLine("  Start time: ", summary.StartTime.ToString("u"));
-                writer.WriteLabelLine("    End time: ", summary.EndTime.ToString("u"));
-                writer.WriteLabelLine("    Duration: ", string.Format(NumberFormatInfo.InvariantInfo, "{0:0.000} seconds", summary.Duration));
+            writer.AppendUICultureFormattedNumber($"  Test Count: ", summary.TestCount);
+            writer.AppendUICultureFormattedNumber($", Passed: ", summary.PassCount);
+            writer.AppendUICultureFormattedNumber($", Failed: ", summary.FailedCount);
+            writer.AppendUICultureFormattedNumber($", Warnings: ", summary.WarningCount);
+            writer.AppendUICultureFormattedNumber($", Inconclusive: ", summary.InconclusiveCount);
+            writer.AppendUICultureFormattedNumber($", Skipped: ", summary.TotalSkipCount);
```

Comment:
Why all the $"...." when we are not using any parameters. For clarity I think that we should remove those $ from line 36-56.